CONTRIBUTING.md

```diff
@@ -258,3 +258,18 @@ the main branch. The workflow file should be named as
 [![NuGet version badge](https://img.shields.io/nuget/v/{your_package_name})](https://www.nuget.org/packages/{your_package_name})
 [![NuGet download count badge](https://img.shields.io/nuget/dt/{your_package_name})](https://www.nuget.org/packages/{your_package_name})
 ```
+
+### Guidance for target frameworks
+
+Components shipped from this repo SHOULD comply with the following requirements:
+
+* MUST support the [.NET
+  versions](https://github.com/open-telemetry/opentelemetry-dotnet/tree/main#supported-net-versions)
+  which are supported by the OpenTelemetry main repo packages.
+* MUST not support out-of-support .NET runtimes (eg: `.NET 452`, `.NET Core
```

Comment:
```suggestion
* Not support out-of-support .NET runtimes (e.g.: `.NET Framework 4.5.2`, `.NET Core
```

Use the names from https://learn.microsoft.com/en-us/lifecycle/products/microsoft-net-framework.